themes/aframe/source/js/examples.js

```diff
@@ -156,6 +156,15 @@ function init () {
     this.setAttribute('href', getNextNavLink());
   });
 
+  var replayLink = document.querySelector('#exampleReplay');
+  if(replayLink) {
+    replayLink.addEventListener('click', function(e) {
+      e.preventDefault();
+      var exampleIframe = document.querySelector('#exampleIframe');
```

Comment:
do we have to re-query this every time?

I would [remove the variable assignment below](https://github.com/darkwing/aframe-site/blob/e10c43e/themes/aframe/source/js/examples.js#L172) and move this line to above line 159.